### PR TITLE
IDP Initiate POST SLO Implementation

### PR DESCRIPTION
**Pull request by gurtzoo**
IDP Initiate POST SLO Implementation

### CODE REVIEW

**Comment by gurtzoo** [comment]
Hi, i have remove all the commits relatives to the white spaces.
Should be correct for validation.
thank you.

**Comment by gurtzoo** [comment]
Hi, can we have an update on this?

**Comment by simo5** [comment]
@gurtzoo superficially it looks good to me, but unfortunately I do not have the time needed to test it and merge it. Hopefully someone else can do that and post their comments and results here.

**Comment by gurtzoo** [comment]
@thijskh Can you validate this PR please?

**Comment by thijskh** [comment]
I can have a look, but since SLO is not really widely in use in practice there's no abundance of testing infrastructure.

**Comment by gurtzoo** [comment]
Hi, yes you are right, on my side i have deployed locally an SP(Apache/Mellon) and test it with an IDP and validate the SLO POST directly.

File: auth_mellon_handler.c [hunk 1]
```diff
@@ -191,6 +191,9 @@ static char *am_generate_metadata(apr_pool_t *p, request_rec *r)
    <SingleLogoutService\n\
      Binding=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\"\n\
      Location=\"%slogout\" />\n\
+   <SingleLogoutService\n\
+     Binding=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\"\n\
+     Location=\"%slogout\" />\n\
```
Comment (thijskh) [inline auth_mellon_handler.c:196]: This is added with `%s` but an extra variable is not added in r215. `/mellon/metadata` now gives internal server error.

**Review by thijskh — APPROVED** [review]
Sorry for the lack of time to review this earlier. I've tested the code and logout works, the metadata generation is broken, see comment. Given the long time this has taken and the simple fix I will fix metadata generation directly after merging this. Thanks!

**Comment by gurtzoo** [comment]
Thank you @thijskh for the validation and correction, i appreciate.

**Pull request merged**